activerecord/src/main/scala/ActiveRecord.scala

```diff
@@ -57,6 +57,10 @@ trait ActiveRecordBase[T] extends CRUDable with ActiveModel with ActiveRecord.As
     this
   }
 
+  def reload: Option[this.type] = {
+    recordCompanion.find(id)
```

Comment:
And it can be a bit confusing because it's immutable.
```scala
val newRecord = record.reload
// record is unchanged and not equal to newRecord
```